scripts/check-and-run-apollo-cli.sh

```diff
@@ -1,7 +1,7 @@
 # Only major and minor version should be specified here
 REQUIRED_APOLLO_CLI_VERSION=1.9
-# Only major version should be specified here
-REQUIRED_NODE_VERSION=8
+# Specify fully qualified version here
+REQUIRED_NODE_VERSION=8.0.0
```

Comment:
I think sticking with the major is probably better; especially considering that 8.0.0 is EOL and only the latest 8 (8.15.0 at this writing) is considered LTS.

Reply:
Ah good context. Thanks @ljharb. Nodenv was requiring my to specify a fully qualified version. I’ll update this to be 8.15.0 since that is LTS.